xml/System/String.xml

```diff
@@ -4466,10 +4466,18 @@ Depending on Why you want to call the `Copy` method, there are a number of alter
         <Parameter Name="value" Type="System.Char" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">The character to compare to the character at the end of this instance.</param>
+        <summary>Determines whether the end of this string instance matches the specified character.</summary>
+        <returns><see langword="true" /> if <paramref name="value" /> matches the end of this instance; otherwise, <see langword="false" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This method performs a word (case-sensitive and culture-sensitive) comparison using the current culture.
```

Comment:
This method performs case-sensitive and culture-sensitive comparison using the current culture.